_maps/shuttles/cargo_birdboat.dmm

```diff
@@ -155,7 +155,7 @@
 /turf/closed/wall/mineral/titanium,
 /area/shuttle/supply)
 "w" = (
-/turf/template_noop,
+/turf/open/space,
```

Comment:
You should figure out the problem with noop because that's the correct turf here

Reply:
What ninja said about the noop